arches_lingo/src/arches_lingo/themes/lingo_theme.ts

```diff
@@ -0,0 +1,119 @@
+import { definePreset, palette } from "@primeuix/themes";
+import { ArchesPreset, DEFAULT_THEME } from "@/arches/themes/default.ts";
+import type { archesPreset } from "@/arches_lingo/types.ts";
+
+const lingoColors = Object.freeze({
+    lightBlueGray: "#ebeef0",
+    lightGray: "#ebeef0",
+    gray: "#dddddd",
+});
+
+const archesPrimitives = ArchesPreset.primitive as archesPreset;
+
+export const LingoPreset = definePreset(ArchesPreset, {
+    primitive: {
+        ...archesPrimitives,
+        ...lingoColors,
+        arches: {
+            legacy: {
+                sidebarDark: "#b9c9d9", // TODO: use shade from surface palette or pick better color
+            },
+        },
+    },
+    semantic: {
+        focusRing: {
+            width: "0.35rem",
+            style: "solid",
+            color: "{primary-color}",
+            offset: "0.05rem",
+            borderRadius: "0.05rem",
+        },
+        colorScheme: {
+            light: {
+                primary: palette(archesPrimitives.arches.blue),
+                surface: palette("{slate}"),
+                sidenav: {
+                    background: "{arches.legacy.sidebar}",
+                },
+                // header: {
+                //     background: "{surface.50}",
+                // },
+                // footer: {
+                //     background: "{surface.50}",
+                // },
+                sortAndFilterControls: {
+                    background: "{light-blue-gray}",
+                    border: "{gray}",
+                },
+                checkbox: {
+                    background: "{light-gray}",
+                },
+            },
+            dark: {
+                // DRW: develop Arches blue color for dark mode
+                primary: palette(archesPrimitives.arches.blue),
```

Comment:
why are we going with a blue color instead of the standard?

Reply:
Is this a complaint about the background colors having blue hues? If so, I set the surface to use zinc rather than slate which has blue hues (controlled by `surface`, not `primary`).

Reply:
I guess, why is the default dark mode palette being overridden at all?

Reply:
resolving for now -- iterative process and all